local-setup/jupyter/crs_loader.py

```
@@ -0,0 +1,1363 @@
+
+        # Check if tenant exists
+        search_url = f"{self.base_url}/mdms-v2/v1/_search"
+        search_payload = {
+            "MdmsCriteria": {
+                "tenantId": "pg",
+                "moduleDetails": [{"moduleName": "tenant", "masterDetails": [{"name": "tenants"}]}]
+            },
+            "RequestInfo": {"apiId": "Rainmaker"}
+        }
+
+        resp = requests.post(search_url, json=search_payload, headers={"Content-Type": "application/json"})
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)